e2e/main_test.go

```diff
@@ -90,23 +94,28 @@ func TestMain(m *testing.M) {
 		}
 	}
 
-	// Install vault-operator
-	testenv.Setup(installVaultOperator)
-	testenv.Finish(uninstallVaultOperator, envfuncs.DeleteNamespace("vault-operator"))
+	if bootstrap {
+		// Install vault-operator
+		testenv.Setup(installVaultOperator)
+		testenv.Finish(uninstallVaultOperator, envfuncs.DeleteNamespace("vault-operator"))
 
-	testenv.Setup(envfuncs.CreateNamespace("vault-secrets-webhook"), installVaultSecretsWebhook)
-	testenv.Finish(uninstallVaultSecretsWebhook, envfuncs.DeleteNamespace("vault-secrets-webhook"))
+		testenv.Setup(envfuncs.CreateNamespace("vault-secrets-webhook"), installVaultSecretsWebhook)
+		testenv.Finish(uninstallVaultSecretsWebhook, envfuncs.DeleteNamespace("vault-secrets-webhook"))
 
-	// Set up test namespace
-	// ns := envconf.RandomName("webhook-test", 16)
-	// testenv.Setup(envfuncs.CreateNamespace(ns))
-	// testenv.Finish(envfuncs.DeleteNamespace(ns))
+		// Set up test namespace
+		// ns := envconf.RandomName("webhook-test", 16)
+		// testenv.Setup(envfuncs.CreateNamespace(ns))
+		// testenv.Finish(envfuncs.DeleteNamespace(ns))
 
-	// Unsealing and Vault access only works in the default namespace at the moment
-	testenv.Setup(useNamespace("default"))
+		// Unsealing and Vault access only works in the default namespace at the moment
```

Comment:
Should we consider adding the feature to support different namespaces (maybe even on cluster-wide level)?

Reply:
It's actually just misconfiguration in the policies. Yes, tests should be able to run in random namespaces, but it's out of scope for this PR.

Reply:
#161